django/templatetags/static.py

```diff
@@ -150,5 +156,14 @@ def do_static(parser, token):
     return StaticNode.handle_token(parser, token)
 
 
+@register.simple_tag
 def static(path):
+    """
+    Return ether a StaticNode or staticfiles storage url if configured.
+
+    :param path: relative path to static asset
```

Comment:
Well what style do we use? I couldn't any reference on how to style the signature documentation. I can do it the google way if you want me to.